packages/web-components/src/components/ic-side-navigation/ic-side-navigation.css

```diff
@@ -11,6 +10,7 @@
     --ic-transition-duration-slow
   );
   --side-navigation-width: 320px;
+  z-index: var(--ic-z-index-overlay);
```

Comment:
Could we create a new z-index variable for side nav? This will help us get started with https://github.com/mi6/ic-ui-kit/issues/391